src/applications/discharge-wizard/tests/v2/cypress/discharge-upgrade-wizard.cypress.spec.js

```diff
@@ -8,7 +8,7 @@ xdescribe('Discharge Upgrade Wizard', () => {
       cy.visit(`${h.ROOT}/introduction1`);
 
       // Home
-      h.verifyUrl(`${ROUTES.HOME}1`);
+      h.verifyUrl(`${ROUTES.HOME}`);
```

Comment:
I think we could just use `h.verifyUrl(ROUTES.HOME);` here to simplify the syntax a little. Looks like we have this in a bunch of spots in this PR.